crates/api_models/src/payment_methods.rs

```diff
@@ -634,14 +634,12 @@ pub struct TokenizedBankValue1 {
     pub bank_routing_number: Option<String>,
     pub bic: Option<String>,
     pub bank_sort_code: Option<String>,
-    pub blz: Option<String>,
-    pub bank_transit_number: Option<String>,
+    pub iban: Option<String>,
```

Comment:
mask it with secret

Reply:
Resolved in a separate commit - https://github.com/juspay/hyperswitch/pull/1384/commits/ac41daafceda596d06c428eb4b15eb15fadd337b